src/app/pages/buy/components/onramp-provider-layout.tsx

```diff
@@ -9,19 +9,19 @@ import { BuyTokensSelectors } from '@tests/page-objects/buy-tokens-selectors';
 const providersInfo = {
   moonpay: {
     title: 'MoonPay',
-    body: 'Purchase STX with credit card, debit card, bank transfer, Apple or Google pay.',
+    body: 'Available for US and non-US residents',
```

Comment:
I don't understand why we'd use this language. `US and non-US residents === everyone`, I flagged it before as bizarre phrasing.

I understand that US residents are often excluded from financial platforms, but couldn't we use phrasing that doesn't group the rest of the world under as a "non-US resident"?

_Available for everyone, including US residents_ or words to that effect?

Reply:
The idea here is to reassure the bulk of our current user base (which lives in the US) that this option is suitable for them.

I agree the wording is a bit weird, but it's the best I can think of that puts "US" in front so that it likely catches the user's eye. 

"Available for everyone, including US residents" buries the lede a bit, though if you feel strongly, we can run it and see how the numbers perform.

Reply:
Why do we even need text here if it is available for everyone? Can't the Okcoin option just say `Only available for non-US residents` while the others just don't have text here at all?

Reply:
Maybe "Available for residents worldwide" is better than "Available for US and non-US residents"?

Having text for these others may be helpful in case someone wonders about coverage in the absence of information ("It says non-US residents only for this one, but nothing for this one. Maybe coverage is the same?")